### DISCUSSION TITLE
What characters are allowed in a VARCHAR column?

### DISCUSSION THREAD

**Discussion by Sebastian-Nielsen** [planetscale/discussion#516] [FAQs] [answered]
What set of characters are allowed in a VARCHAR column?

**Comment by jgreet** [comment, marked as answer]
The default character set for all databases on PlanetScale is [utf8mb4](https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-utf8mb4.html). The other supported character sets are listed [in our docs](https://planetscale.com/docs/reference/mysql-compatibility#overview).

See also: https://planetscale.com/blog/mysql-charsets-collations
